README.md

```diff
@@ -1,120 +1,61 @@
-### A Wix Events and Wix eCommerce Next.js Music Tour Template
+# A Wix Events and Wix eCommerce Next.js Music Tour Template
 
-![](docs/media/template-showcase.gif)
+![Template showcase](docs/media/template-showcase.gif)
 
 This template is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app). It uses [Wix Headless](https://dev.wix.com/api/sdk/about-wix-headless/overview) to leverage the Wix Events and Wix Stores business solutions for managing event ticketing and a store.
 
-## Part I: Get started
+## Local Development
 
-To integrate the Wix Stores and Wix Events business solutions with the template, first create and set up a project on Wix:
+Prerequisites:
 
-### Step 1: Create a project on Wix
+1. [Create a Wix Headless project](https://dev.wix.com/docs/go-headless/getting-started/setup/general-setup/create-a-project)
+2. [Add the Stores and Events apps to your project](https://dev.wix.com/docs/go-headless/getting-started/setup/general-setup/add-apps-to-a-project)
+3. Authorize the template with [quick start deployment](https://manage.wix.com/headless-funnel-nextjs/netlify?repository=https://github.com/wix/wix-commerce-ticketing-nextjs-template) or by [creating an OAuth app](https://dev.wix.com/docs/go-headless/getting-started/setup/authorization/create-an-o-auth-app-for-visitors-and-members)
```

Comment:
This should be `https://manage.wix.com/headless-funnel-nextjs/select-platform?repository=https://github.com/wix/wix-commerce-ticketing-nextjs-template`
(`select-platform` instead of `netlify`)